apps/src/templates/studioHomepages/CourseBlocksGradeBands.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import ContentContainer from '../ContentContainer';
+import ResourceCard from './ResourceCard';
+import styleConstants from '../../styleConstants';
+import i18n from "@cdo/locale";
+
+const pegasusContentWidth = styleConstants['content-width'];
+
+const styles = {
+  container: {
+    width: pegasusContentWidth,
+    display: "flex",
+    justifyContent: "space-between"
+  }
+};
+
+const CourseBlocksGradeBands = React.createClass({
```

Comment:
I was planning to as soon as I had [the React 15.5.4 PR](https://github.com/code-dot-org/code-dot-org/pull/16861) ready to go, with tests passing and everything.  Not quite there yet.